packages/common/src/utils/stream-utils.ts

```diff
@@ -280,7 +280,10 @@ export class StreamUtils {
     ipfs: IpfsApi
   ): Promise<CeramicCommit> {
     if (StreamUtils.isSignedCommit(commit)) {
-      const block = await ipfs.block.get(toCID((commit as DagJWS).link))
+      const block = await ipfs.block.get(toCID((commit as DagJWS).link), {
```

Comment:
Can we change this to go through the Dispatcher that has this logic already, instead of using the raw `ipfs` handle?

Reply:
I don't think that is easily possible. The dispatcher lives in core and this code is inside common. Currently core depends on common so this would create a cyclic dependency. Seems like a good refactor to move this logic to the dispatcher but for now its seems out of scope for this simple change.